doc-content/enterprise-only/project-data/preview-editor-delete-test-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='preview-editor-delete-test-proc']
+= Deleting a test scenario
+
+You can delete existing test scenarios which were created using the *Test Scenario (Preview)* editor.
+
+.Procedure
+. From the *Test Scenario (Preview)* editor toolbar on the upper-right, click *Delete*.
+. In the *Confirm Delete* window,
+* If you wish to add a comment regarding the deletion of the test scenario, click *add a comment*.
+* Click *Delete*.
+
+You will see a message stating that the test scenario was deleted successfully.
```

Comment:
"A message stating that the test scenario was deleted successfully appears on the screen."
There are more occurrences of the result statements in this doc.

Reply:
There were 4 instances of such highlighted result statements. All changed as suggested.